episodes/chap5_python_client.md

```diff
@@ -35,75 +73,352 @@ server_api = "/api"
 username = "MY USERNAME"
 password = "MY PASSWORD"
 
-organization_key = "" # Path to the encryption key, if encryption is enabled.
+# Path to the private key, if encryption is enabled. Can be None if
+# not used. Note that this key is the organization's private key. In
+# case of this workshop we do not use encryption, so this can be None.
+organization_key = None
 ```
 
-Once you have created the Python module with the configuration settings, you can create the client instance as follows:
+Once you have created the Python module with the configuration settings, you can import it and create the client instance as follows:
 
 ```python
-from vantage6.client import UserClient as Client
+# client.py
 
-# Note: we assume here the config.py you just created is in the current directory.
-# If it is not, then you need to make sure it can be found on your PYTHONPATH
+from vantage6.client import Client
+
+# It is assumed here that the `config.py` you just created is in the current
+# directory. If it is not, then you need to make sure it can be found on
+# your PYTHONPATH
 import config
 
-# Initialize the client object, and run the authentication
+# Initialize the client object, and authenticate
 client = Client(config.server_url, config.server_port, config.server_api,
                 log_level='debug')
 client.authenticate(config.username, config.password)
+# In the case of 2FA, you should also include the 6-digit code:
+# client.authenticate(config.username, config.password, '123456')
```

Comment:
should we mention this?

Reply:
Why not? I think it is nice to show them then in case they connect to a real server they probably have to do an additional step.

Reply:
I agree. This is also good as this material will likely be used as a reference in the future.